src/librustdoc/clean/types.rs

```diff
@@ -2081,6 +2081,7 @@ crate enum ImportKind {
 crate struct ImportSource {
     crate path: Path,
     crate did: Option<DefId>,
+    crate attrs: Option<Attributes>,
```

Comment:
I would store this in a `Box` too to avoid making ImportSource larger than it has to be (Attributes is 72 bytes large).
```suggestion
    crate attrs: Option<Box<Attributes>>,
```